test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -45,12 +52,12 @@ export async function openMultichainDappAndConnectWalletWithExternallyConnectabl
 }
 
 /**
- * Sends a request to wallet extension to create session for the passed scopes.
+ * Initiates a request to wallet extension to create session for the passed scopes.
  *
  * @param driver - E2E test driver {@link Driver}, wrapping the Selenium WebDriver.
  * @param scopes - scopes to create session for.
  */
-export async function createSessionScopes(
+export async function initCreateSessionScopes(
```

Comment:
I feel like the previous name was a bit more accurate. Perhaps going simpler would be better? `createCaip25Session`, `createMultichainSession`, `createSession` even? What are your thoughts here?

Reply:
I see why you renamed this given the confirmation has been moved out of this helper. I guess let's just leave it as is

Reply:
I wanted to add the `init` here for explicitness, because this will not create the scope all the way through. It will initiate the interaction with the wallet UI to send request for it, but will not finish by pressing the `confirm` button. Hence the `init`, so the caller explicitly knows it will not conclude the call. Does that make sense ?

Reply:
yes! maybe `initCreateSessionApproval` in that case?

Reply:
nevermind!